experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -71,10 +72,18 @@ export class NodeSDK {
   private _meterProvider?: MeterProvider;
   private _serviceName?: string;
 
+  private _disabled?: boolean;
+
   /**
    * Create a new NodeJS SDK instance
    */
   public constructor(configuration: Partial<NodeSDKConfiguration> = {}) {
+    if (getEnv().OTEL_SDK_DISABLED) {
+      this._disabled = true;
+      // Finish the instantiation of SDK object
```

Comment:
I find this line of comment misleading. It doesn't early return.

Reply:
I updated the comment based on your feedback. 
You find it more clear? 

Thanks!